src/Sarif.Viewer.VisualStudio/ErrorList/ErrorListService.cs

```diff
@@ -176,11 +176,28 @@ public static void ProcessLogFile(string filePath, Solution solution, string too
                 log = JsonConvert.DeserializeObject<SarifLog>(logText);
             }
 
-            ProcessSarifLog(log, outputPath, solution);
+            ProcessSarifLog(log, outputPath, solution, showMessageOnNoResults: promptOnLogConversions);
 
             SarifTableDataSource.Instance.BringToFront();
         }
 
+        /// <summary>
+        /// Closes the specified SARIF log in the viewer.
+        /// </summary>
+        /// <param name="paths">The complete path to the SARIF log file.</param>
+        public static void CloseSarifLogs(IEnumerable<string> paths)
```

Comment:
Just a question.  Should this move to uri's?

Reply:
The existing APIs use strings, so I wanted to keep that pattern.

---
In reply to: [440942501](https://github.com/microsoft/sarif-visualstudio-extension/pull/143#discussion_r440942501) [](ancestors = 440942501)